magicctapipe/lstchain/reco/utils.py

```diff
@@ -0,0 +1,23 @@
+import numpy as np
+
+
+def add_delta_t_key(events):
```

Comment:
again, same function as in lstchain 0.10

Reply:
See comment on the lstchain __init__ file